docs/build/blast-install-1.md

```diff
@@ -39,31 +29,13 @@ Checkout [Setup Rust](/docs/build/setup-rust)
 
 ## 02 Install Cudos Blast
 
-### Globally - Create a new project anywhere
-
-Use `npm` to install Cudos Blast CLI tool globally. This means you can create a new project anywhere. 
-
-```bash
-npm install cudos-blast -g
-
-# confirm installation
-
-blast --version
```

Comment:
This doesn't actually work. If you're not "in a Node project" - i.e. if the current directory or an ancestor contains a package.json - then most blast commands don't work, not even `blast --version` or `blast --help`.

The only command that works is `blast init`, which initializes a Blast project in the current directory.